### DISCUSSION TITLE
The preLaunchTask 'C/C++: clang build active file' terminated with exit code -1.

### DISCUSSION THREAD

**Discussion by Harrylam0410** [microsoft/vscode-cpptools#9701] [Q&A]
What is that mean? I am new to programming.
I am just trying to run a simple code following CS50.
Why did it happen and how should I solve it?
Thank you so much!
<img width="1440" alt="Screenshot 2022-08-08 at 1 27 11 AM" src="https://user-images.githubusercontent.com/110656567/183345553-108ffc05-dccd-4f07-bdca-8ba7b88f354a.png">

**Comment by Colengms** [comment]
Hi @Harrylam0410 .  It looks like you're trying to 'build and run' the active file, but the code appears to have failed to build.  The error dialog is indicating that the build failed and is asking whether you want to run/debug anyway (if there is a previously successfully built executable present).  I see the error in the terminal window in your image; It appears to have failed to find `_get_int` when linking.

**Reply by yickysan** [reply]
So how does one fix this error because I ran into similar issues.
